homeassistant/components/landisgyr_heat_meter/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Landis+Gyr Heat Meter",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/landisgyr_heat_meter",
-  "requirements": ["ultraheat-api==0.4.1"],
+  "requirements": ["ultraheat-api==0.4.2"],
```

Comment:
This should be a separate PR (one for the timeout, and one for the dependency bump)

Also, it seems a type hint has been removed in the package. Was it a mistake?
https://github.com/vpathuis/uh50/compare/v0.4.1...v0.4.2#diff-29c719d84b7ad7d0034426cd294e872518fe1261c91cdb76b615074ca181d42bR19

Reply:
Ah okay, didn't know that, thanks. I'll remove the dependency upgrade for now, since it's only logging and the timeout is more important.
The type hint was wrong so I removed it. But I'll set it to the correct type hint in stead (tuple).

Reply:
I've removed the dependency upgrade from this PR and changed the PR-description accordingly.